alonzo/impl/test/lib/Test/Cardano/Ledger/Alonzo/Serialisation/Generators.hs

```diff
@@ -134,7 +134,11 @@ instance Mock c => Arbitrary (Tx (AlonzoEra c)) where
       <*> arbitrary
 
 instance Mock c => Arbitrary (Script (AlonzoEra c)) where
-  arbitrary = frequency [(1, pure PlutusScript), (9, NativeScript <$> arbitrary)]
+  arbitrary =
+    frequency
+      [ -- (1, pure PlutusScript) -- Until we have one of these we never generate one.
```

Comment:
maybe use the word `TODO` so we don't lose track of this?

Reply:
I am hoping to get things updated to the new version of plutus, so I can resolve this.

Reply:
I added a TODO, will try and get the newer version of Plutus compilng.

Reply:
OK, with alwaysSucceeds and alwaysFails I occaisionally generate one of them, so now all possible Scripts are covered.